app-modules/form/src/Models/FormEmailAutoReply.php

```diff
@@ -92,4 +104,15 @@ public function getMergeData(Student|Prospect|null $author): array
             'student preferred name' => $author->getAttribute($author->displayPreferredNameKey()),
         ];
     }
+
+    /**
+     * @todo Remove this dynamic cast once `RefactorEngagementCampaignSubjectToJsonb` is removed.
+     *       Move 'subject' casting to the static `$casts` array: protected $casts = ['subject' => 'array'];
+     */
+    protected function casts(): array
```

Comment:
Does this overwrite the `$casts` property? If so, we will need to make sure that the other casts are in here as well.